Examples/test-suite/octave/Makefile.in

```diff
@@ -36,6 +38,11 @@ CPP_TEST_BROKEN += \
 #C_TEST_CASES += \
 #	complextest
 
+# Tests which are broken when compiled with Clang
+ifeq (yes,$(CLANGXX))
+FAILING_MULTI_CPP_TESTS = mod template_typedef_import
+endif
```

Comment:
This is too much extra makefile machinery for compiler specific problems for my liking. I'd rather you used the approach we generally use for compiler specific problems in tests which is to use detect a specific compiler (+ version thereof if necessary) and add in appropriate workarounds. I don't know what the specific problem is and whether this is possible, but for now, leave octave in the .travis `allow_failures`. I'll have a look then and perhaps find a solution. If it is a compiler problem, a newer version of xcode may work (try `osx_image: xcode10.2` ) in .travis.yml for this octave build.

Reply:
I managed to find a fix for the two tests (`mod`, `template_typedef_import`) that were failing on OSX, so the Octave OSX build should now pass. With the pre-compilation speed-up it should also hopefully be reliably completed within the Travis timeout, and so I've removed it from `allow_failures`.